Gopkg.lock

```diff
@@ -475,6 +474,23 @@
   revision = "4b7aa43c6742a2c18fdef89dd197aaae7dac7ccd"
   version = "1.0.1"
 
+[[projects]]
+  digest = "1:7226c7e52dcd9273715f24d6e3f023857ccfc955c2bb09fcd9fea733b6cbb96f"
+  name = "github.com/openshift-knative/knative-serving-operator"
```

Comment:
We don't want this, since this was the old repo location.

Reply:
OK